### PR TITLE
fix: [CI-6862]: re-run in debug mode feature flag

### PR DESCRIPTION
**Pull request by shaurya-harness**
### Summary

re-run in debug mode feature flag

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by vardanbansal-harness — APPROVED** [review]

**Review by d1wilko — DISMISSED** [review]
LGTM

**Review by kapilkumawat86 — APPROVED** [review]

**Review by ishantmehtaharness — DISMISSED** [review]
Looks Good

**Comment by shaurya-harness** [comment]
retrigger jest

**Comment by shaurya-harness** [comment]
retrigger coverage

File: src/modules/10-common/featureFlags.ts [hunk 1]
```diff
@@ -143,6 +143,7 @@ export enum FeatureFlag {
   SPG_SIDENAV_COLLAPSE = 'SPG_SIDENAV_COLLAPSE',
   PL_REMOVE_EXTERNAL_USER_ORG_PROJECT = 'PL_REMOVE_EXTERNAL_USER_ORG_PROJECT',
   PIPELINE_ROLLBACK = 'PIPELINE_ROLLBACK',
+  CI_REMOTE_DEBUG = 'CI_REMOTE_DEBUG'
```
Comment (d1wilko) [inline src/modules/10-common/featureFlags.ts:146]: this needs a comma at the end I think

**Review by d1wilko — CHANGES_REQUESTED** [review]
1 small change :)

File: src/modules/10-common/featureFlags.ts (hunk 1 above)
Comment (shaurya-harness) [reply src/modules/10-common/featureFlags.ts]: yes, updated

**Review by d1wilko — APPROVED** [review]
LGTM

**Pull request merged**